beacon-chain/state/state-native/field_trie.go

```diff
@@ -100,20 +115,23 @@ func (f *FieldTrie) RecomputeTrie(indices []uint64, elements interface{}) ([32]b
 	if err := f.validateIndices(indices); err != nil {
 		return [32]byte{}, err
 	}
+	if val, ok := elements.(SliceAccessor[*BeaconState]); ok {
+		f.numOfElems = val.Len(val.State())
```

Comment:
If you have a struct ( which is what we use), calling Len() on it will panic. From the docs of `reflect` package:
```
// Len returns v's length.
// It panics if v's Kind is not Array, Chan, Map, Slice, String, or pointer to Array.
```